crates/node-core/src/init.rs

```diff
@@ -297,6 +300,7 @@ pub fn insert_genesis_header<DB: Database>(
 pub fn init_from_state_dump<DB: Database>(
```

Comment:
this needs more docs, not obvious what this does

Reply:
Added the following. Does it help?
```rust
/// Reads account state from a [`BufRead`] reader and initializes it at the highest block that can
/// be found on database.
///
/// It's similar to [`init_genesis`] but supports importing state too big to fit in memory, and can
/// be set to the highest block present. One practical usecase is to import OP mainnet state at
/// bedrock transition block.
pub fn init_from_state_dump<DB: Database>(
```